libs/remix-ui/tree-view/src/lib/tree-view-item/tree-view-item.tsx

```diff
@@ -13,7 +13,7 @@ export const TreeViewItem = (props: TreeViewItemProps) => {
 
   return (
     <li ref={innerRef} key={`treeViewLi${id}`} data-id={`treeViewLi${id}`} className='li_tv' {...otherProps}>
-      <div key={`treeViewDiv${id}`} data-id={`treeViewDiv${id}`} className={`d-flex flex-row align-items-center ${labelClass}`} onClick={() => !controlBehaviour && setIsExpanded(!isExpanded)}>
+      <div key={`treeViewDiv${id}`} data-id={`treeViewDiv${id}`} className={`d-flex flex-row align-items-center ${labelClass}`} style={{border: "1px solid transparent"}} onClick={() => !controlBehaviour && setIsExpanded(!isExpanded)}>
```

Comment:
Cool, how about creating an atomic class called `border-transparent`? Or another suggested name? Thank.

Reply:
you could add this in "li_tv" class we have here https://github.com/ethereum/remix-project/blob/3e28a6df517400630d6a422e03b666cebdb34055/libs/remix-ui/tree-view/src/lib/remix-ui-tree-view.css

Reply:
Yeah Ok

Reply:
Hi @LianaHus, thanks. But I just tried, seems not to be working. Am I adding to the correct position? The root cause of this issue is the hover border CSS, but I didn't see any border settings in `.li_tv`.

<img width="974" alt="image" src="https://user-images.githubusercontent.com/95468177/233052536-a42b98cf-2b94-4d66-a18a-3cebb23c9c95.png">

Reply:
Hi @LianaHus, any new ideas? From my perspective, using an inline style is the best way in this case, and I saw some other places are using the inline style. Otherwise, please suggest a class name for it. I think the `transparent-border` might be a good one.